spec/honeycomb/integrations/redis_spec.rb

```diff
@@ -2054,7 +2054,17 @@
 
         let(:redis) do
           allow(connection).to receive(:read).and_return(slots, nodes, commands)
-          Redis.new(driver: driver, cluster: cluster, replica: true).tap do
+          Redis.new(
+            driver: driver, cluster: cluster, replica: true,
+          ).tap do |redis|
+            # https://github.com/redis/redis-rb/issues/1017
+            # establishes connections to all nodes in the cluster,
+            # otherwise extra READONLY commands are executed on replicas
```

Comment:
Nit: my head parsed this comment a little strangely. I thought it was saying that the code in the GitHub issue established a connection. Maybe reverse the phrasing?
```suggestion
            # Establishes connections to all nodes in the cluster,
            # otherwise extra READONLY commands are executed on replicas
            # due to https://github.com/redis/redis-rb/issues/1017
```